code/modules/clothing/suits/miscellaneous.dm

```diff
@@ -419,7 +419,7 @@
 	item_state = "labcoat"
 	body_parts_covered = CHEST|GROIN|ARMS
 	cold_protection = CHEST|GROIN|ARMS
-	min_cold_protection_temperature = FIRE_SUIT_MIN_TEMP_PROTECT
+	min_cold_protection_temperature = 5

```

Comment:
temperatures are in Kelvin, are you sure you want the wintercoat to protect against temperatures up to -268 celsius?
You probably mean `= T0C + 5` (5 degrees above zero celsius)